bundles/org.openhab.binding.meater/README.md

```diff
@@ -0,0 +1,56 @@
+# Meater Binding
+
+This is an openHAB binding for the Meater probe, by Apption Labs.
+
+This binding uses the Meater cloud REST API.
+
+![Meater+ Probe](doc/meater-plus-side.png)
+
+## Supported Things
+
+This binding supports the following thing types:
+
+- meaterapi: Bridge - Communicates with the Meater cloud REST API.
+
+
+- meaterprobe: The Meater probe - Only support for cloud connected Meater probes (Meater+) 
+
+## Discovery
+
+The preferred way of adding Meater probe(s) since the probe IDs are not easily found.
+
+**NOTE**: You need to have your Meater probe(s) connected to the cloud and the Meater app running before you start the discovery.
+
+After the configuration of the Bridge, your Meater probe(s) will be automatically discovered and placed as a thing(s) in the inbox.
```

Comment:
Perhaps it should be mentioned if the Meater app is still needed after discovery?

Reply:
OK, I can add that it is necessary to have the app running during the whole cook.

Reply:
In the meantime I investigated how it works - and placed an order for Meater Block. :-) So now I'm wondering: Is it even required to have the app running for discovery of Meater Block? If not, perhaps it could be mentioned that this requirement is for The Original Meater and Meater Plus.

So maybe: "**NOTE**: For The Original Meater and Meater Plus you need to have your Meater charger connected to the cloud through the Meater app before you start the discovery."

Reply:
Since I do not own a Meater Block I haven't tested that. For Meater Plus that I own, it is necessary to have the app running for both discovery and during cook. I'm not sure why it should differ between them though.

Reply:
For Meater Block the charger is Wi-Fi connected, so only Bluetooth between probe and charger.

Reply:
I have a Meater block and I can confirm that the app is not needed, as long as the device is connected to both WiFi and the cloud (not standalone mode). I was able to discover the 4 probes, but had to do a manual scan to get them to appear.

Reply:
OK, thanks for the info, I will update the README accordingly.